files.js/modal.js

```diff
@@ -0,0 +1,165 @@
+    html: 'html',
+    bootstrap: 'bootstrap',
+    ruby: 'Ruby On Rails',
+    projectImage: 'images/Snapshoot Portfolio.png',
+    projectDesc: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam amet animi voluptate doloribus at quaerat, vitae dicta sunt possimus voluptatum, facilis vel? Eligendi, harum ipsum excepturi sint at quasi? Repellendus eveniet suscipit quibusdam inventore architecto minus eligendi adipisci. Harum atque voluptate dolore fugiat pariatur eius ullam architecto porro aspernatur nihil.',
+    projectbtnlive: 'https://carlylechia.github.io/MY-PORTFOLIO/',
+    projectbtnsource: 'https://github.com/carlylechia/MY-PORTFOLIO.git',
+  },
+];
+
+function modalpopup() {
+  popUp.style.display = 'block';
+  body.appendChild(popUp);
+  const showing = `
+  <div class="popupcont">
+      <div class="popupcontheader">
+        <h2 class="desktophide">${projectList[projectList.length - 1].title}</h2>
+        <h2 class="mobilehide">Keeping track of hundreds  of components website</h2>
+        <button id="closeModal" class="closeModal" draggable = "true">${projectList[projectList.length - 1].closebtn}</button>
+      </div>
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
+        </ul>
+      </div>
+      <div class="popupcontbody">
+        <div class="popupcontbodyleft">
+          <img src="${projectList[projectList.length - 1].projectImage}" alt="image">
+        </div>
+        <div class="popupcontbodyright">
+          <p class="desktophide">${projectList[projectList.length - 1].projectDesc}</p>
+          <p class="mobilehide">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+          <div class="popupcontbodyrightbtns">
+          <a href="https://carlylechia.github.io/MY-PORTFOLIO/"><button type="button" id="modalbtns live">See Live <img src="images/viewlive.png" alt="view live"></button></a>
+          <a href="https://github.com/carlylechia/MY-PORTFOLIO.git"><button type="button" id="modalbtns live">See Source <img src="images/viewsource.png" alt="view Source"></button></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  `;
+  popUp.innerHTML = showing;
+  body.appendChild(popUp);
+
+  document.addEventListener('click', (e) => {
+    if (e.target.id === 'closeModal') {
+      popUp.style.display = 'none';
+    }
+  });
+}
```

Comment:
- [**OPTIONAL**] What do you think of making our popup properly responsive on the desktop sizes?. Let's check out how it looks on your design and Figma design
   
  

     Your Implementation | Figma Design
    :---:|:---:
    ![](https://user-images.githubusercontent.com/63087491/152171176-5d8971f4-ed92-4f75-9c62-4eca73c62ae6.png) | ![](https://user-images.githubusercontent.com/63087491/152171193-cfd336d9-bedd-4a57-a091-1588a8ba8257.png)

- Did you notice how your buttons float over the text?. Please let's try to fix this.

Reply:
Thank you @FrederickMih . I have adjusted it